alluxiofs/core.py

```diff
@@ -120,6 +116,17 @@ def __init__(
             if preload_path is not None:
                 self.alluxio.load(preload_path)
 
+        if "log_level" in test_options:
+            log_level = test_options["log_level"].upper()
+            if log_level == "DEBUG":
+                logger.setLevel(logging.DEBUG)
+            elif log_level == "INFO":
+                logger.setLevel(logging.INFO)
+            elif log_level == "WARN" or log_level == "WARNING":
+                logger.setLevel(logging.WARN)
+            else:
+                logger.warning(f"Unsupported log level: {log_level}")
```

Comment:
and lets set a default warning level to be ERROR (logging has ERROR level correct? )

Reply:
logging usually don't have error, if it's error, the program should directly error out and failed
but they do have error log level